WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.wordpress.android.util;
+
+import android.app.Activity;
+import android.net.Uri;
+
+import org.wordpress.android.analytics.AnalyticsTracker;
+import org.wordpress.android.fluxc.model.SiteModel;
+import org.wordpress.android.ui.prefs.SiteSettingsInterface;
+
+public class WPMediaUtils {
+    public static Uri getOptimizedMedia(Activity activity, SiteModel siteModel, String path, boolean isVideo) {
+        boolean isOptimized = false;
+        if (!NetworkUtils.isWiFiConnected(activity) && !isVideo) {
```

Comment:
Should this also be skipped when the network type is `TYPE_ETHERNET` or `TYPE_WIMAX`?